src/scene/shader-lib/chunks/standard/frag/frontend.js

```diff
@@ -0,0 +1,37 @@
+export default /* glsl */`
+struct Frontend {
+    vec2 uvOffset;
```

Comment:
what is uvOffset - this does not feel self-explanatory. I think we should add comments for all members here.